docs/readthedocs/source/doc/Orca/QuickStart/orca-pytorch-quickstart.md

```diff
@@ -71,43 +62,53 @@ class LeNet(nn.Module):
         x = self.fc2(x)
         return F.log_softmax(x, dim=1)
 
-model = LeNet()
-model.train()
 criterion = nn.NLLLoss()
-adam = torch.optim.Adam(model.parameters(), 0.001)
+```
+After defining your model, you need to define a *Model Creator Function* that returns an instance of your model, and a *Optimizer Creator Function* that returns a PyTorch optimizer.
+
+```python
+def model_creator(config):
+    model = LeNet()
+    return model
+
+def optim_creator(model, config):
+    return torch.optim.Adam(model.parameters(), lr=0.001)
 ```
 
 ### Step 3: Define Train Dataset
 
-You can define the dataset using standard [Pytorch DataLoader](https://pytorch.org/docs/stable/data.html). 
+You can define the dataset using a *Data Creator Function* that returns a PyTorch `DataLoader`. Orca also supports [Orca SparkXShards](../Overview/data-parallel-processing).
```

Comment:
Need to align the contents with the notebook.
Notebook it write like this `You can define the dataset using a Data Creator Function that has two parameters config and batch_size and returns a PyTorch DataLoader.`